packages/domains-table/src/primary-domain-label/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Badge, Gridicon, Popover } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { useRef, useState, ComponentRef } from 'react';
+import './style.scss';
+
+export const PrimaryDomainLabel = () => {
+	const [ showPrimaryDomainInfo, setShowPrimaryDomainInfo ] = useState( false );
+	const { __ } = useI18n();
+	const popoverRef = useRef< ComponentRef< typeof Badge > >( null );
+
+	return (
+		<Badge
+			ref={ popoverRef }
+			className="domains-table__primary-domain-label"
+			type="info-green"
+			onMouseEnter={ () => setShowPrimaryDomainInfo( true ) }
+			onMouseLeave={ () => setShowPrimaryDomainInfo( false ) }
+		>
+			{ __( 'Primary domain' ) }
```

Comment:
When using `__()` in packages one has to pass in a domain parameter. It's annoying that there's no eslint warning for it.

```suggestion
			{ __( 'Primary domain', __i18n_text_domain__ ) }
```

In the Calypso app `__i18n_text_domain__` is just transformed into `'default'`. But if the package is imported into another app (e.g. ETK) the placeholder gives that other app an opportunity to set the domain to something else.